plugins/ssr/ssrprinters.ml

```diff
@@ -74,12 +74,19 @@ let pr_term (k, c) = pr_guarded (guard_term k) pr_glob_constr_and_expr c
 let pr_hyp (SsrHyp (_, id)) = Id.print id
 let pr_hyps = pr_list pr_spc pr_hyp
 
+let is_empty = function
+  | [] -> true
+  | _ -> false
+
```

Comment:
There is otherwise a function `List.is_empty` which does already that.

Reply:
Ah yes, I actually missed that, because it turns out to be `CList.is_empty`. Not very discoverable :-)

Reply:
But we have a trick... File `util.ml` rebinds `CList` to `List`, while `CList` includes OCaml's  `List`, so, we can transparently use the `List` prefix w/o having to care whether the function comes from OCaml stdlib or from our extension of it. So, the `C` is not needed!

Reply:
Yes, I saw that, but in this file `Util` is not imported, so autocomplete did not suggest it :-)

Reply:
Ah, ok, too bad.